modules/core/src/v2/coins/dot.ts

```diff
@@ -221,6 +210,27 @@ export class Dot extends BaseCoin {
     return { txHex: signedTxHex };
   }
 
+  /**
+   * Explain/parse transaction
+   * @param params
+   * @param callback
+  */
+  async explainTransaction(params: ExplainTransactionOptions): Promise<accountLib.Dot.TransactionExplanation> {
+    const txHex = params.txPrebuild.txHex;
```

Comment:
there might be an issue here if `params.txPrebuild` is undefined - maybe move this below the invalid params check?